### DISCUSSION TITLE
Feature Request: Add a "Name" field for remote Servers

### DISCUSSION THREAD

**Discussion by Omegatcu** [spatie/ray#420] [Ideas]
Hello,

I would like to have a Name field for the Remote Servers, I have multiple servers I'm working on and to remember which IP is which server is pretty hard (Usernames are mostly the same or totally random in case of CloudWays), so a Textfield would help with that a lot.

**Comment by freekmurze** [comment]
That makes sense, we'll add this!

**Reply by Omegatcu** [reply]
Thank you very much!

**Reply by S-R0** [reply]
@freekmurze Any idea on a release date for this?
